e2e/jest.test-setup.ts

```diff
@@ -128,45 +143,69 @@ beforeEach(async () => {
   });
 
   page.on('requestfinished', async (request) => {
-    if (canLogResponse(request)) {
-      if (isApiFailure(request)) {
-        await logError(request);
-      } else {
-        await logResponse(request);
+    let method;
+    let url;
+    let status;
+    try {
+      if (canLogResponse(request)) {
+        // Try find out what the request was if exception thrown.
+        method = request.method();
+        const resp = request.response();
+        url = resp.url();
+        status = resp.status();
+
+        if (isApiFailure(request)) {
+          await logError(request);
+        } else {
+          const responseBody = await transformResponseBody(request);
+          if (skipApiResponseBody(request)) {
+            console.debug('❗ Request finished: ' +
+               `${request.response().status()} ${request.method()} ${request.url()}`);
+          } else {
+            console.debug('❗ Request finished: ' +
+               `${request.response().status()} ${request.method()} ${request.url()}\n${responseBody}`);
+          }
+        }
       }
+    } catch (err) {
+      // Try find out what the request was
+      console.error(`${err}\n${status} ${method} ${url}`);
```

Comment:
Exception could be thrown when parsing long or badly formatted string. Simply print out the request info to let me know what request was when errored.
```
Exception occurred when getting console messages.
      Error: Protocol error (Runtime.callFunctionOn): Object reference chain is too long

      153 |       // tslint:disable-next-line:no-empty
      154 |     } catch (err) {
    > 155 |       console.error(`Exception occurred when getting console messages.\n${err}`);
          |               ^
      156 |     }
      157 |   });
```